pyi.py

```diff
@@ -1019,12 +1019,16 @@ def visit_Call(self, node: ast.Call) -> None:
 
     # 3.8+
     def visit_Constant(self, node: ast.Constant) -> None:
-        if not self.string_literals_allowed.active and isinstance(node.value, str):
+        if (
+            node.value
```

Comment:
nit: this condition would read better to me if it was moved to after narrowing node.value to str